content/docs/about/kubeflow.md

```diff
@@ -11,6 +11,44 @@ recreate other services, but to provide a straightforward way to deploy
 best-of-breed open-source systems for ML to diverse infrastructures. Anywhere 
 you are running Kubernetes, you should be able to run Kubeflow.
 
+## Getting started with Kubeflow
+
+Follow the [getting-started guide](/docs/started/getting-started) to set up your
+environment.
+
+Then read the [documentation](/docs/) to learn how to use Kubelow.
+Highlights include the following:
+
+* Kubeflow includes services for spawning and managing 
+  [Jupyter notebooks](/docs/notebooks/). [Project Jupyter](https://jupyter.org/) 
+  is a non-profit, open source project that supports interactive data science 
+  and scientific computing across many programming languages.
+
+* [Kubeflow Pipelines](/docs/pipelines/pipelines-overview/) is a platform for 
+  building, deploying, and managing multi-step ML workflows based on Docker 
+  containers.
+
+* Kubeflow offers a number of [components](/docs/components/) that you can use
+  to build your ML training, hyperparameter tuning, and serving workloads across
+  multiple platforms.
+
+## What is Kubeflow?
+
+Kubeflow is *the machine learning toolkit for Kubernetes*.
+
+To use Kubeflow, the basic workflow is:
+
+* Download and run the Kubeflow deployment binary.
+* Customize the resulting configuration files.
+* Run the specified scripts to deploy your containers to your chosen 
```

Comment:
```suggestion
* Run the specified scripts to deploy your containers to your specific 
```

Reply:
Done - changed the word "chosen" to "specific".